### PR TITLE
[PD-1205] Fixed off by one on readable data

### PR DESCRIPTION
**Pull request by aspidites**
Because you know, I've not been programming for years in languages where array-like structures are 0-indexed...

### CODE REVIEW

File: static/reporting.163-8.js [hunk 1]
```diff
@@ -375,10 +375,11 @@ Report.prototype.readable_data = function() {
 
         if (key === "contact type") {
           value = $.map(checklists.contact_type, function(item, index) {
-            if (value.indexOf(item.value) > 0) {
+            if (value.indexOf(item.value) !== -1) {
               return item.label;
             }
           });
+          console.log(value);
```
Comment (galitskyd) [inline static/reporting.163-8.js]: console.log

**Comment by almathew** [comment]
Do the version numbers of these static files need updated?

**Comment by aspidites** [comment]
@almathew def.jobs probably, reporting, probably not.

**Comment by aspidites** [comment]
Actually, i just removed a console log statement from def.ui.js so I really don't think it matters either if that's cached. I remember why I didn't bother updating the name now.

**Comment by almathew** [comment]
Works for me.

**Pull request merged**